gradle/scripts/globalDependencies.gradle

```diff
@@ -57,6 +57,7 @@ subprojects {
         }
       }
       all*.exclude group: 'org.apache.calcite', module: 'calcite-avatica' // replaced by org.apache.calcite.avatica:avatica-core
+      all*.exclude group: 'org.mockito', module: 'mockito-all'
```

Comment:
Can we leave a comment why we need to make this exclusion?

Reply:
Actually. I don't think we need this. There is no transitive on mockito-all dependency in the latest iteration (maybe there was when I was drafting the PR). I am going to remove this